### PR TITLE
Add error handler for 500 errors.

### PR DESCRIPTION
**Pull request by easternbloc**
This includes custom content for session_timeout errors.

### CODE REVIEW

File: locales/en/default.json [hunk 1]
```diff
@@ -561,5 +561,15 @@
       "country": "Current location",
       "date-lost": "Date realised"
     }
+  },
+  "errors": {
+    "default": {
+      "title": "Sorry, something went wrong",
+      "message": "A server exception occured. Please try again."
```
Comment (JoeChapman) [inline locales/en/default.json]: s/occured/occurred

Comment (easternbloc) [inline locales/en/default.json]: oops

**Comment by JoeChapman** [comment]
:+1: nice work

File: app.js [hunk 2]
```diff
@@ -86,6 +86,8 @@ app.use(initSession);
 
 app.use(require('./routes'));
 
+app.use(require('./errors/'));
```
Comment (lennym) [inline app.js]: What is this trailing slash? It makes me feel uneasy.

Comment (easternbloc) [inline app.js]: It's a thing but it should probably be removed for clarity.

**Pull request merged**